koku/masu/external/downloader/oci/oci_report_downloader.py

```diff
@@ -302,13 +302,14 @@ def get_manifest_context_for_date(self, date):
             invoice_month = month.strftime("%Y%m")
             assembly_id = ":".join([str(self._provider_uuid), str(invoice_month)])
             month_file_names = self._extract_names(assembly_id, month)
-            manifest_id = self._process_manifest_db_record(assembly_id, str(month), len(month_file_names), dh._now)
-            monthly_report["manifest_id"] = manifest_id
-            monthly_report["assembly_id"] = assembly_id
-            monthly_report["compression"] = manifest_dict.get("compression")
             files_list = [{"key": key, "local_file": self.get_local_file_for_report(key)} for key in month_file_names]
-            monthly_report["files"] = files_list
-            report_manifests_list.append(monthly_report)
+            if len(files_list) > 0:
```

Comment:
A truthy check is equivalent and more idiomatic.
```suggestion
            if files_list:
```